net/nheko/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           qt5 1.0
+
+github.setup        mujx nheko 0.4.3 v
+categories          net chat
+platforms           darwin
+license             GPL-3
+maintainers         {sireesh @scarface-one gmail.com:sireeshkodali1}
+description         A matrix chat client
+long_description    Nheko is a native desktop matrix \
+                    chat client. It aims to be more \
+                    like a chat app and less like an \
+                    IRC client
+
+patchfiles          patch-SideBarAction.h.diff \
+                    patch-RoomInfoListItem.cc.diff \
+                    patch-TopRoomBar.cc.diff
+
+checksums           rmd160  075e4b41d690a1557f4a0f8136f89494d0a7b51e \
+                    sha256  1db6ef979fb9711f728b9d0640790ad4a5ee6613a5f7caaf3e12e6c370a63038 \
+                    size    4461579
+
+configure.args-append   -H.\
+                    -Bbuild \
+                    -DCMAKE_BUILD_TYPE=Release \
+                    -DCMAKE_INSTALL_PREFIX=.
+
+depends_lib         port:qt5-qttools \
+                    port:qt5-qtmultimedia \
+                    port:qt5-qtsvg \
+                    port:qt5-qtmacextras \
+                    port:cmake \
+                    port:fontconfig \
+                    port:lmdb \
+                    port:boost \
+                    port:libsodium
+
```

Comment:
Please use `depends_lib-append` to not overwrite portgroup dependencies and sort the dependencies by name. 

Overall, once Perry's changes are also implemented I think this looks good.

Reply:
fixed